api/grpc/mpi/v1/common.proto

```diff
@@ -34,6 +34,8 @@ message CommandResponse {
         COMMAND_STATUS_IN_PROGRESS = 3;
         // Command failure
         COMMAND_STATUS_FAILURE = 4;
+        // Command status of upload has completed
+        COMMAND_STATUS_UPLOAD_COMPLETE = 5;
```

Comment:
I thought that sending back a `COMMAND_STATUS_OK` would indicate that a config upload is complete?